Sources/LogtoSocialPlugin/LogtoSocialPlugin.swift

```diff
@@ -30,12 +30,17 @@ public struct LogtoSocialPluginConfiguration {
 public protocol LogtoSocialPlugin {
     var connectorId: String { get }
     var urlSchemes: [String] { get }
+    var isAvailable: Bool { get }
 
     func handle(url: URL) -> Bool
     func start(_ configuration: LogtoSocialPluginConfiguration)
 }
 
 public extension LogtoSocialPlugin {
+    var isAvailable: Bool {
+        true
```

Comment:
should be `true`? maybe `false`?

Reply:
default should be true

Reply:
unless needed, wechat sdk is awful

Reply:
why default true?

Reply:
See Slack [thread](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1649907465208309?thread_ts=1649906599.502909&cid=C02A8G4HVAM)